### PR TITLE
Add Secondary Staging Smoke Tests

### PR DESCRIPTION
**Pull request by camdesgov**
### WHAT
 Add new `SUBDOMAIN` environment variable to smoke tests to make it easier to adapt the tests to different environments in future (e.g. production, staging, development etc)

Comment out Slack notifications in Secondary Staging environment for now. We will re-enable them when work on the Secondary Staging environment is complete.

### WHY
We need these to ensure that the apps and infrastructure in the Secondary Staging environment are functioning correctly.

Link to Trello card (if applicable): https://trello.com/c/bUZn21mb/1443-story-get-concourse-smoke-tests-working-with-secondary-account

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,12 @@ This is a collection of rspec with Capybara tests, using Firefox to access the l
 Running the tests
 -----------------
 
-The tests run in a headless browser inside a Docker container. You need to provide some environment variables:
+The tests run in a headless browser inside a Docker container. You need to provide some environment. variables:
```
Comment (colinbm) [inline README.md:9]: Rogue full stop?

File: smoke-tests-concourse.yml [hunk 2]
```diff
@@ -186,8 +187,71 @@ jobs:
...
+      - name: properties
+      params:
+        DOCKER: docker
+        GW_USER: "((SECONDARY_STAGING_GW_USER))"
+        GW_PASS: "((SECONDARY_STAGING_GW_PASS))"
+        GW_2FA_SECRET: "((SECONDARY_STAGING_GW_2FA_SECRET))"
+        GOOGLE_API_CREDENTIALS: "((GOOGLE_API_CREDENTIALS))"
+        GOOGLE_API_TOKEN_DATA: "((GOOGLE_API_TOKEN_DATA))"
+        RADIUS_KEY: "((RADIUS_KEY))"
+        RADIUS_IPS: "((RADIUS_IPS))"
```
Comment (colinbm) [inline smoke-tests-concourse.yml:252]: Should these have `SECONDARY_STAGING_` prefixes on the radius vars?

File: smoke-tests-concourse.yml [hunk 3]
```diff
@@ -186,8 +187,71 @@ jobs:
...
+        SUBDOMAIN: "staging-temp.wifi"
+    # on_failure:
+    #   put: notify
+    #   params:
+    #     text_file: properties/failure_message
```
Comment (colinbm) [inline smoke-tests-concourse.yml:257]: Do we want the alerts back now?

File: spec/system/signup/shared_context.rb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 RSpec.shared_context "signup", shared_context: :metadata do
   before(:all) do
-    unless ENV["GOOGLE_API_CREDENTIALS"] && ENV["RADIUS_KEY"] && ENV["RADIUS_IPS"]
-      abort "\e[31mMust define GOOGLE_API_CREDENTIALS, RADIUS_KEY, and RADIUS_IPS\e[0m"
+    unless ENV["GOOGLE_API_CREDENTIALS"] && ENV["RADIUS_KEY"] && ENV["RADIUS_IPS"] && ENV["SUBDOMAIN"]
+      abort "\e[31mMust define GOOGLE_API_CREDENTIALS, RADIUS_KEY, SUBDOMAIN and RADIUS_IPS\e[0m"
     end
```
Comment (colinbm) [inline spec/system/signup/shared_context.rb:5]: `SUBDOMAIN` between the two `RADIUS_` vars?

File: smoke-tests-concourse.yml (hunk 2 above)
Comment (camdesgov) [reply smoke-tests-concourse.yml]: Good spot! I've updated this now and added the correct creds to the smoke tests pipeline

File: spec/system/signup/shared_context.rb (hunk 4 above)
Comment (camdesgov) [reply spec/system/signup/shared_context.rb:5]: I've swapped these round now so it's more readable.

File: smoke-tests-concourse.yml (hunk 3 above)
Comment (camdesgov) [reply smoke-tests-concourse.yml:257]: I was going to leave these commented out just for the secondary staging smoke tests for the time being, as I'm still occasionally breaking things. They are still on for both the staging and production pipelines.

File: README.md (hunk 1 above)
Comment (camdesgov) [reply README.md]: Ah yes! This has now been deleted.

**Review by colinbm — APPROVED** [review]

**Pull request merged**